code/run_templates/filter1/etl.py

```diff
@@ -0,0 +1,92 @@
+import shutil
+import os
+import sys
+import cosmotech_api
+from cosmotech_api.model.dataset import Dataset
+from pathlib import Path
+
+sys.path.append(str(Path(__file__).parents[1] / "etl_sub_dataset_by_filter"))
+
+from twingraph import (
+    parse_twingraph_json,
+    create_csv_files_from_graph_content,
+    upload_twingraph_zip_archive,
+)
+from common import get_logger, get_api
+
+LOGGER = get_logger()
+
+
+def main():
+    LOGGER.info("Starting the ETL Run")
+    organization_id = os.environ.get("CSM_ORGANIZATION_ID")
+    workspace_id = os.environ.get("CSM_WORKSPACE_ID")
+    runner_id = os.environ.get("CSM_RUNNER_ID")
+    api = get_api()
+
+    runner = api["runner"].get_runner(
+        organization_id=organization_id, workspace_id=workspace_id, runner_id=runner_id
+    )
+    subdataset_id = runner["dataset_list"][0]
+    parent_dataset_id = runner["dataset_list"][1]
```

Comment:
Since this script will be used as an example, maybe we could add a comment here to explain that these indices in `dataset_list` are defined by a webapp convention, for subdataset creation scripts?